magenta/models/nsynth/wavenet/generate.py

```diff
@@ -0,0 +1,146 @@
+import tensorflow as tf
+from scipy.io import wavfile
+from magenta.models.nsynth import utils
+from magenta.models.nsynth.wavenet.h512_bo16 import Config, FastGenerationConfig
+import numpy as np
+
+
+def inv_mu_law(x, mu=255.0):
```

Comment:
These functions should probably be loaded from / added to utils.py. You could just rename them as inv_mu_law_numpy() for example.